postgres/datadog_checks/postgres/statements.py

```diff
@@ -191,8 +203,8 @@ def _load_pg_stat_statements(self, db):
             params=params,
         )
 
-    def _collect_metrics_rows(self, db):
-        rows = self._load_pg_stat_statements(db)
+    def _collect_metrics_rows(self, db, tags):
+        rows = self._load_pg_stat_statements_safe(db, tags)
```

Comment:
The `_safe` suffix doesn't seem necessary to me

Reply:
I thought it looked a little better to split the method that does the logic from the one that handles the errors which is why I needed a suffix for the latter. That said, this can all just be in `_load_pg_stat_statements` so that's what the latest version is doing.